src/richie/apps/core/fields/multiselect.py

```diff
@@ -147,15 +147,15 @@ def from_db_value(value, *_args):
         """Convert a database value to a list value."""
         if not value:
             return None if value is None else []
-        return list([v.strip() for v in value.split(",")])
+        return list((v.strip() for v in value.split(",")))
```

Comment:
Maybe `return (v.strip() for v in value.split(","))` is enough and a generator is ok?

Reply:
Good question! From my understanding, the list casting is volunteer here to see clearly what is returned by this method as before the comprehension list was already wrap in a list constructor.
@sampaccoud WDYT?

Reply:
Yes I would say the "from_db_value" is what is used to populate the python value of the field. So if you don't return a list it will not be possible to interact with the field as expected to e.g. add a value to the list...